### PR TITLE
All Public Pages

### PR DESCRIPTION
**Pull request by taimoormalik97**

### CODE REVIEW

File: app/controllers/users/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  # def new
+  #   super
+  # end
+
+  # POST /resource/confirmation
+  # def create
+  #   super
+  # end
+
+  # GET /resource/confirmation?confirmation_token=abcdef
```
Comment (itratzahra7) [inline app/controllers/users/confirmations_controller.rb]: remove commented code

File: app/controllers/users/passwords_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  layout 'static_pages_layout', only: [:new, :edit]
+  # GET /resource/password/new
+  # def new
+  #   super
+  # end
+
+  # POST /resource/password
+  # def create
+  #   super
+  # end
```
Comment (itratzahra7) [inline app/controllers/users/passwords_controller.rb]: remove commented code

File: app/controllers/users/registrations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  # before_action :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  # def new
+  #   super
+  # end
+
+  # POST /resource
+  # def create
+  #   super
```
Comment (itratzahra7) [inline app/controllers/users/registrations_controller.rb]: remove

File: app/controllers/users/sessions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  layout 'static_pages_layout', only: [:new]
+  # before_action :configure_sign_in_params, only: [:create]
+
+  # GET /resource/sign_in
+  # def new
+  #   super
+  # end
+
+  # POST /resource/sign_in
+  # def create
```
Comment (itratzahra7) [inline app/controllers/users/sessions_controller.rb]: remove

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<footer class="py-4 nav-static-pages text-white-50 fixed-bottom">
+  <div class="container text-center">
+    <small><%= link_to 'Home', root_path, class: "text-white-50" %></small> |
+    <small><%= link_to 'About', about_path, class: "text-white-50" %></small> |
```
Comment (itratzahra7) [inline app/views/layouts/_footer.html.erb]: move to en

File: app/views/layouts/static_pages_layout.html.erb [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    <nav class="navbar nav-static-pages navbar-expand-lg navbar-light fixed-top">
+        <a class="navbar-brand text-white ml-3" href= <%= root_path %> > HMS</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item">
+              <a class="nav-link text-white" href= <%= about_path %> >About</a>
```
Comment (itratzahra7) [inline app/views/layouts/static_pages_layout.html.erb]: en

File: app/views/public_pages/index.html.erb [hunk 7]
```diff
@@ -1,10 +1,116 @@
...
+<html>
+  <head>
+    <title>Hospital</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <script>
+      $(document).scroll(function () {
+        var $nav = $(".nav-transparent");
```
Comment (itratzahra7) [inline app/views/public_pages/index.html.erb]: move this toh js file
remove $

File: app/controllers/public_pages_controller.rb [hunk 8]
```diff
@@ -1,8 +1,22 @@
 class PublicPagesController < ApplicationController
-  layout false
+  layout 'static_pages_layout', only: [:about, :contact, :find]
 
   # GET /resource/index
   def index
+    respond_to do |format|
+      format.html { render layout: false }
```
Comment (sulemanuzair) [inline app/controllers/public_pages_controller.rb:7]: can we set layout: false at controller start similar to other actions

File: app/controllers/users/passwords_controller.rb [hunk 9]
```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Users::PasswordsController < Devise::PasswordsController
+  layout 'static_pages_layout', only: [:new, :edit]
+  # GET /resource/password/new
+  # def new
```
Comment (sulemanuzair) [inline app/controllers/users/passwords_controller.rb]: can we set layout for particular actions in in application controller ?

File: app/javascript/scss/_custom.scss [hunk 10]
```diff
@@ -86,3 +86,202 @@ $color-primary-btn: #007bff;
...
+  color: #ccc;
+  content: open-quote;
+  font-size: 4em;
+  line-height: 0.1em;
+  margin-right: 0.25em;
+  vertical-align: -0.4em;
+}
+
+blockquote p {
+display: inline;
```
Comment (sulemanuzair) [inline app/javascript/scss/_custom.scss]: space

File: app/javascript/scss/_custom.scss [hunk 11]
```diff
@@ -86,3 +86,202 @@ $color-primary-btn: #007bff;
...
+}
+
+.nav-transparent.scrolled {
+  background-image: none !important;
+  background-color: #5e72e4 !important;
+  box-shadow: 0px 5px 3px rgba(0, 0, 0, 0.2)
+}
+
+.main-sign-up-btn {
+  background-color: #5e72e4;
```
Comment (MirahSaqib) [inline app/javascript/scss/_custom.scss]: make variables for colors

File: app/javascript/scss/_custom.scss [hunk 12]
```diff
@@ -86,3 +86,202 @@ $color-primary-btn: #007bff;
...
+  vertical-align: -0.4em;
+}
+
+blockquote p {
+display: inline;
+}
+
+.hear-reviews {
+  margin-top: 50px;
+  color: DimGrey !important;
```
Comment (MirahSaqib) [inline app/javascript/scss/_custom.scss]: variable

Comment (MirahSaqib) [reply app/javascript/scss/_custom.scss]: and use color code

File: app/views/public_pages/contact.html.erb [hunk 13]
```diff
@@ -0,0 +1,57 @@
+<% provide(:title, 'Contact') %>
+<h1 class="mt-5 pt-5 mb-4 text-center">Contact Us</h1>
+<div class="col-md-12">
+  <div class="row public_pages_without_links">
+    <div class="col-md-6 h-100 ml-5">  
+      <h4> Complaints & Queries</h4>
```
Comment (MirahSaqib) [inline app/views/public_pages/contact.html.erb]: en.yml

File: app/views/public_pages/index.html.erb [hunk 14]
```diff
@@ -1,10 +1,116 @@
...
+          <h1 class="welcome-h1">Hospital Management System</h1>
+          <% if request.subdomain.empty? %>
+            <%= link_to "Sign up", new_user_registration_path, class: "btn btn-md sign-up-button" %>
+          <% end %>
+        </div>
+      </div>
+    </div>
+    <div>
+      <div id="demo" class="carousel slide" data-ride="carousel">
+        <div class="text-center"><h1 class="hear-reviews">Hear from Our Doctors</h1></div>
```
Comment (MirahSaqib) [inline app/views/public_pages/index.html.erb]: internationalization

**Pull request merged**